recipes/qt/6.x.x/conanfile.py

```diff
@@ -518,6 +518,14 @@ def build(self):
     def _cmake_executables_file(self):
         return os.path.join("lib", "cmake", "Qt6Core", "conan_qt_executables_variables.cmake")
 
+    @property
```

Comment:
I think this could be more future proof by making it a simple function taking the module (core, qml, etc) as a parameter

Reply:
Something like this?

Reply:
no, EXACTLY this ;)